app/tests/integration/nimbus/remote_settings/pages/login.py

```diff
@@ -0,0 +1,34 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Login(Page):
+
+    _root_locator = (By.CSS_SELECTOR, ".container-fluid")
+    _auth_type_checkbox_locator = (
+        By.CSS_SELECTOR,
+        "div.radio:nth-child(2) label span input",
```

Comment:
`:nth-child()` scares me, but my guts tell me this might be the RS login page which is probably not worth trying to do upstream PRs to add better selectors (unless we need to automate remote settings for multiple projects).

Reply:
Yeah, I may do that though in the future.